### PR TITLE
Minor tweaks to Mods stubs

### PR DESCRIPTION
**Pull request by paper-toaster**
Did a quick spellcheck on all articles, fixed some weird sentences wherever I happened to notice, and added a piece of trivia to the Auto mod stub

### CODE REVIEW

File: wiki/Game_modifier/Auto/en.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ The mod functions exactly like a [replay](/wiki/Replay), with the options for 0.
 
 ### osu!standard
 
-In [osu!standard](/wiki/Game_mode/osu!), when encountered with a spinner, the Auto mod will spin to a maximum of 477 spins per minute (SPM) in a counter-clockwise rotation.
+In [osu!standard](/wiki/Game_mode/osu!), when encountered with a spinner, the Auto mod will spin to a maximum of 477 spins per minute (SPM) in a counterclockwise rotation.
```
Comment (MegaApplePi) [inline wiki/Game_modifier/Auto/en.md:39]:
```suggestion
In [osu!standard](/wiki/Game_mode/osu!), when encountered with a spinner, the Auto mod will spin to a maximum of 477 spins per minute (SPM) in a anticlockwise rotation.
```

File: wiki/Game_modifier/Co-op/en.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ tags:
 
 *Notice: Enabling the Co-op mod will result in an unranked play.*
 
-The **Co-op** mod is a [game modifier](/wiki/Game_modifier) for [osu!mania](/wiki/Game_mode/osu!mania) that introduces a second playfield with a second set of controls based on the current *x*K count (7K -> 7K + 7K = 14K). For osu!mania-specific [beatmaps](/wiki/Beatmaps), the *x*K is split to half (7K -> 4K + 3K = 7K).
+The **Co-op** mod is a [game modifier](/wiki/Game_modifier) for [osu!mania](/wiki/Game_mode/osu!mania) that introduces a second playfield with a second set of controls based on the current *x*K count (e.g., 7K -> 7K + 7K = 14K). For osu!mania-specific [beatmaps](/wiki/Beatmaps), the *x*K is split to half (e.g., 7K -> 4K + 3K = 7K).
```
Comment (TPGPL) [inline wiki/Game_modifier/Co-op/en.md:32]:
```suggestion
The **Co-op** mod is a [game modifier](/wiki/Game_modifier) for [osu!mania](/wiki/Game_mode/osu!mania) that introduces a second playfield with a second set of controls based on the current *x*K count (e.g. 7K -> 7K + 7K = 14K). For osu!mania-specific [beatmaps](/wiki/Beatmaps), the *x*K is split to half (e.g. 7K -> 4K + 3K = 7K).
```

File: wiki/Game_modifier/Flashlight/en.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ When paired with the Hidden mod, the fruits are momentarily visible if the catch
 
 ### osu!mania
 
-In [osu!mania](/wiki/Game_mode/osu!mania), the visible area is restricted to a relatively thin horizontal bar at the center of the screen whilst everything else is obstructed from view. Thus this could be thought of the Hidden mod and [Fade In](/wiki/Game_modifier/Fade_In) mod combined. (That is, without the changing sizes of the visible area.)
+In [osu!mania](/wiki/Game_mode/osu!mania), the visible area is restricted to a relatively thin horizontal bar at the center of the track whilst everything else is obstructed from view. In this respect, the Flashlight mod could be thought of as the Hidden mod and [Fade In](/wiki/Game_modifier/Fade_In) mod combined. (That is, without the changing sizes of the visible area.)
```
Comment (TPGPL) [inline wiki/Game_modifier/Flashlight/en.md:64]:
```suggestion
In [osu!mania](/wiki/Game_mode/osu!mania), the visible area is restricted to a relatively thin horizontal bar at the center of the track whilst everything else is obstructed from view. In that sense, the Flashlight mod could be thought of as the Hidden mod and [Fade In](/wiki/Game_modifier/Fade_In) mod combined. (That is, without the changing sizes of the visible area.)
```

File: wiki/Game_modifier/Half_Time/en.md [hunk 4]
```diff
@@ -40,13 +40,13 @@ As a result, using the Half Time mod will lead to an increase in maximum possibl
...
 
-In addition to this, the leniency for hyperdashes are increases, making it much to stop underneath the next fruit when doing a hyperdash.
+In addition to this, the leniency for hyperdashes is increased, which makes it much more difficult to stop underneath the next fruit when performing a hyperdash.
 
 ## Trivia
 
 - When the Half Time mod is enabled, the `Length`, `BPM`, and `Objects` values will be colored in a light blue with the new values. (Pictured below.)
   - The `Objects` value will still be colored light blue even though no change in value is actually made.
-- The `CS`, `AR`, and `HP` values will have a superscript triangle next to their values indicate a small decrease in their values. (Pictured below.)
+- The `CS`, `AR`, and `HP` values will have a superscript triangle next to their values that indicate a small decrease in their values. (Pictured below.)
```
Comment (TPGPL) [inline wiki/Game_modifier/Half_Time/en.md:49]:
```suggestion
- The `CS`, `AR`, and `HP` values will have a superscript triangle next to their values that indicates a small decrease in their values (pictured below).
```

File: wiki/Game_modifier/Target_Practice/en.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ tags:
 
 ![TP mod icon](/wiki/shared/mods/TP.png "Target Practice (TP) mod icon")
 
-*For the full list of all mods, see: [Game Modifiers](/wiki/Game_modifier)*\
+*For the full list of all mods, see: [Game Modifiers](/wiki/Game_modifier)*
```
Comment (TPGPL) [inline wiki/Game_modifier/Target_Practice/en.md:18]: now broken without the backslash

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Game_modifier/Auto/en.md [hunk 6]
```diff
@@ -36,11 +36,11 @@ The mod functions exactly like a [replay](/wiki/Replay), with the options for 0.
 
 ### osu!standard
 
-In [osu!standard](/wiki/Game_mode/osu!), when encountered with a spinner, the Auto mod will spin to a maximum of 477 spins per minute (SPM) in a counter-clockwise rotation.
```
Comment (peppy) [inline wiki/Game_modifier/Auto/en.md:39]: i don't understand this change. seemed fine?

**Review by peppy — DISMISSED** [review]

**Review by TPGPL — APPROVED** [review]

File: wiki/Game_modifier/Auto/en.md (hunk 6 above)
Comment (paper-toaster) [reply wiki/Game_modifier/Auto/en.md:39]: Anticlockwise is the British term

**Pull request merged**